src/foundation.cs

```diff
@@ -12140,6 +12140,9 @@ interface NSDateInterval : NSCopying, NSSecureCoding {
 		bool ContainsDate (NSDate date);
 	}
 
+#if XAMCORE_4_0
+	[DisableDefaultCtor] // -init should never be called on NSUnit!
+#endif
```

Comment:
It's better not to be under 4_0
And add manual, empty ctor in another files (all excluded in 4_0) along with an obsolete attribute